symbols/overlay29.yml

```diff
@@ -2290,6 +2290,28 @@ overlay29:
         
         r0: Entity pointer
         return: True if the specified monster can't act because of a status problem, false otherwise.
+    - name: GetMobilityTypeCheckSlip
+      address:
+        EU: 0x2300058
+        NA: 0x22FF62C
+      description: |-
+        Returns the mobility type of a monster species, accounting for STATUS_SLIP.
+        
+        The function also converts MOBILITY_LAVA and MOBILITY_WATER to other values if required.
+        
+        r0: Monster species
+        r1: True if the monster can walk on water
+        return: Mobility type
+    - name: GetMobilityTypeCheckSlipAndLevitate
```

Comment:
It seems misleading to use the word "Levitate" when it doesn't check for the levitate ability.

Reply:
Updated to match `IsFloating`